docs/reusing_the_docker_daemon.md

```diff
@@ -31,3 +37,39 @@ The fix is to update /etc/sysconfig/docker to ensure that minikube's environment
 ```
 
 Remember to turn off the _imagePullPolicy:Always_, as otherwise Kubernetes won't use images you built locally.
+
+Another approach is to enable minikube registry addons and then push images directly into registry. Steps for this approach is as follows:
+
+Ensure that docker is configured to use `192.168.39.0/24` as insecure registry. Refer [here](https://docs.docker.com/registry/insecure/) for instructions.
```

Comment:
is this IP same on windows, mac on different drivers ... ?

Reply:
Will explain how to decide what subnet to use here.